packages/web-components/package.json

```diff
@@ -88,7 +88,8 @@
     "typecheck": "tsc --noEmit -p tsconfig.json",
     "upgrade-carbon": "yarn upgrade-interactive @carbon/web-components @carbon/icon-helpers @carbon/icons --latest --exact",
     "visual-snapshot": "yarn percy storybook:start ./storybook-static",
-    "wca": "web-component-analyzer analyze src --outFile custom-elements.json"
+    "wca": "web-component-analyzer analyze src --outFile custom-elements.json",
+    "cypress:verify": "cypress verify"
```

Comment:
What does this command do?

Reply:
[This command](https://docs.cypress.io/app/references/command-line#cypress-verify) will:

> Verify that Cypress is installed correctly and is executable.

Before we were running into these errors a lot, which were causing the whole test suite to fail:

![image](https://github.com/user-attachments/assets/3d68f14a-c98e-448e-9983-446e108af6d8)

After some googling of the issue, this was the recommended fix. It seems to make the process of finding the cypress executable before the test runs start reliable.